waku/waku_archive/driver/sqlite_driver/queries.nim

```diff
@@ -303,186 +278,13 @@ proc combineClauses(clauses: varargs[Option[string]]): Option[string] =
     where &= " AND " & clause
   return some(where)
 
-proc whereClausev2(
-    cursor: bool,
-    pubsubTopic: Option[PubsubTopic],
-    contentTopic: seq[ContentTopic],
-    startTime: Option[Timestamp],
-    endTime: Option[Timestamp],
-    ascending: bool,
-): Option[string] {.deprecated.} =
-  let cursorClause =
-    if cursor:
-      let comp = if ascending: ">" else: "<"
-
-      some("(storedAt, id) " & comp & " (?, ?)")
-    else:
-      none(string)
-
-  let pubsubTopicClause =
-    if pubsubTopic.isNone():
-      none(string)
-    else:
-      some("pubsubTopic = (?)")
-
-  let contentTopicClause =
-    if contentTopic.len <= 0:
-      none(string)
-    else:
-      var where = "contentTopic IN ("
-      where &= "?"
-      for _ in 1 ..< contentTopic.len:
-        where &= ", ?"
-      where &= ")"
-      some(where)
-
-  let startTimeClause =
-    if startTime.isNone():
-      none(string)
-    else:
-      some("storedAt >= (?)")
-
-  let endTimeClause =
-    if endTime.isNone():
-      none(string)
-    else:
-      some("storedAt <= (?)")
-
-  return combineClauses(
-    cursorClause, pubsubTopicClause, contentTopicClause, startTimeClause, endTimeClause
-  )
-
-proc selectMessagesWithLimitQueryv2(
-    table: string, where: Option[string], limit: uint, ascending = true, v3 = false
-): SqlQueryStr {.deprecated.} =
-  let order = if ascending: "ASC" else: "DESC"
-
-  var query: string
-
-  query =
-    "SELECT storedAt, contentTopic, payload, pubsubTopic, version, timestamp, id, messageHash, meta"
-  query &= " FROM " & table
-
-  if where.isSome():
-    query &= " WHERE " & where.get()
-
-  query &= " ORDER BY storedAt " & order & ", id " & order
-
-  query &= " LIMIT " & $limit & ";"
-
-  return query
-
 proc prepareStmt(
     db: SqliteDatabase, stmt: string
 ): DatabaseResult[SqliteStmt[void, void]] =
   var s: RawStmtPtr
   checkErr sqlite3_prepare_v2(db.env, stmt, stmt.len.cint, addr s, nil)
   return ok(SqliteStmt[void, void](s))
 
-proc execSelectMessagesV2WithLimitStmt(
```

Comment:
Why remove :) ? I guess it has been moved and adapted o another place

Reply:
Yes, legacy code still has that proc.